specification/dartLangSpec.tex

```diff
@@ -6180,7 +6163,7 @@ \subsection{Constants}
   evaluates to an instance of \code{String}.
 
 % New in 2.1.
-\item An expression of the form \code{$e$ as $T$} is potentially constant
+\item An expression of the form \code{$e$\,\,as\,\,$T$} is potentially constant
```

Comment:
Why is this neccessary? It's highly annoying if \code{...} does not provide the correct spacing (mono-spaced preferably). I do not want to write `\,\,` instead of space in code in general, so why is it necessary here?

Reply:
We have 73 occurrences of `\,` in `\code{...}`, generally when the `\code` occurs in text. The reason for having them is that the spacing is not always proportional (in particular `!   nextthing` gets a crazy large space), and also that it's generally confusing that the spaces (even when proportional) are so much larger in `a   code   example` in the middle of a sentence. Let's take that issue separately.